scss/_buttons.scss

```diff
@@ -83,8 +83,10 @@
   &:first-child:active,
   &.active,
   &.show {
-    color: var(--#{$prefix}btn-active-color);
-    background-color: var(--#{$prefix}btn-active-bg);
+    // WILL REMOVING THESE CAUSE OTHER PROBLEMS?
```

Comment:
Obviously can/will remove but I wanted to first enquire to ensure this isn't an ill advised change.